feature/networkinstance/tests/interface_assignments/interface_assignments_test.go

```diff
@@ -0,0 +1,129 @@
+/*
+ Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package interface_assignments
```

Comment:
```suggestion
package interface_assignments_test
```

I think the directory needs to be renamed as well.

Reply:
Fixed.